src/components/CustomerQuota/AddUserModal.tsx

```diff
@@ -156,17 +167,18 @@ export const AddUserModal: FunctionComponent<AddUserModal> = ({
             <Card style={styles.card}>
               <View style={styles.cardHeader}>
                 <AppText style={{ flex: 1 }}>
-                  Add another NRIC to combine customer quotas
+                  Add another ID to combine customer quotas
```

Comment:
I feel like as an optimisation later on, we can dynamically word the type of id in our messages haha. Just one idea!

Reply:
previous Eida was working on standardization for all error messages to have generic words like "ID" instead of NRIC, should have ID here?

Reply:
ohh, that is true haha. Yeah let's leave it as ID then~